### PR TITLE
Bug: fix spacer children height and padding

### PR DESCRIPTION
**Pull request by rgerstenberger**
This fixes a bug where spacer children were affecting page layout.

/cc @alexlande @kenwheeler

### CODE REVIEW

**Comment by alexlande** [comment]
:+1:

File: lib/select.js [hunk 1]
```diff
@@ -393,7 +397,7 @@ classBase.Option = React.createClass({
     this.props.onClick();
   },
   render: function render() {
-    return (
+    return(
```
Comment (kenwheeler) [inline lib/select.js]: does this need a space?

File: lib/select.js [hunk 2]
```diff
@@ -222,16 +222,16 @@ var classBase = React.createClass({
         this.moveIndexByOne( /*decrement*/ev.keyCode === keyboard.upArrow);
         // If not tab, assume alphanumeric
       } else if (ev.keyCode !== keyboard.tab) {
-        this.typeahead(String.fromCharCode(ev.keyCode));
-      }
+          this.typeahead(String.fromCharCode(ev.keyCode));
+        }
```
Comment (kenwheeler) [inline lib/select.js]: TrailingSpaces?

File: lib/select.js (hunk 1 above)
Comment (kenwheeler) [inline lib/select.js]: Oh is this generated? or did you edit directly?

**Comment by kenwheeler** [comment]
:+1:

File: lib/select.js (hunk 1 above)
Comment (rgerstenberger) [inline lib/select.js]: Generated. Not sure why it changed.

**Pull request merged**